### DISCUSSION TITLE
Prev & Next year in Datepicker Month panel

### DISCUSSION THREAD

**Discussion by lasseitorp** [formkit/formkit#1538] [Ideas]
The `sequence` prop is great for as the name implies sequencing the flow of picking a date. Sometimes it can be useful to only want a sole Month-picker, and easy enough `:sequence="['month']` does that job.

Now lets say we need a six-month min/max period, again easy to fix with `min-date` & `max-date`, and disabled months are disabled as expected. The problem arises when a calendar year is broken, now some months are active in one year and some in the next. The Year-button is clickable and can move the user to next year, but for UX consistency it would be nicer to use the `prev` and `next` buttons. Currently the Prev and Next buttons only exist in the` 'day'` and `'year'` panels, but are equally useful in the `'month'` panel.

Thoughts?

**Comment by justin-schroeder** [comment]
This makes perfect sense. Ill promote it to an feature request issue.
